### PR TITLE
Fix live and issue with drm on rpi

### PR DESCRIPTION
**Pull request by Gaet81**
Hello,
I've improved the live and now DRM can be streamed on RPI..it needs to be checked if it works as well on other platform otherwise we'll need to find a way to find the arch to fix it.

For the live, as it has a different url for each live it's no more possible to get it on the PVR...I don't knwo why but when using the PVR it always take the cached json even with the max_cache set to -1.
You need to go to RTBF auvio folder in the live to get it works. If anyone can fix it would be good, otherwise the skeleton file will need to be updated and the channel will need to be deactivated for the live and just leave the RTBF auvio folder.
fix issue #935 https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/issues/935

### CODE REVIEW

**Comment by Gaet81** [comment]
I did hardcode the partner key because it's taking a lot of time to play the stream otherwise

**Comment by darodi** [comment]
works fine. Thanks a lot!

**Comment by darodi** [comment]
- fix #935

File: resources/lib/channels/be/rtbf.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def get_partner_key():
 
 
 # partner_key
-PARTNER_KEY = get_partner_key()
+PARTNER_KEY = '82ed2c5b7df0a9334dfbda21eccd8427' #get_partner_key()
```
Comment (darodi) [inline resources/lib/channels/be/rtbf.py:113]: `PARTNER_KEY = '82ed2c5b7df0a9334dfbda21eccd8427'  # get_partner_key()`

adding a space fixes flak8 error

**Pull request merged**